dashboard/app/models/pd/application/teacher_application.rb

```diff
@@ -125,7 +125,8 @@ def self.next_year(year)
       current_year_index >= 0 ? APPLICATION_YEARS[current_year_index + 1] : nil
     end
 
-    # The census lags by two years, so the census year is the first application year minus 2
+    # The census lags by a year years, so the census year is the first application year minus 1
+    # For example, for the 2023-2025 application year, we want to look at 2023 census data
```

Comment:
Tiny nit (not worth a drone rebuild if you're not already doing another one): duplicate word of "year years" and also "2023-2024" instead of "2023-2025"